app/scripts/lib/setupSentry.js

```diff
@@ -477,6 +478,7 @@ function getClientOptions() {
     beforeBreadcrumb: beforeBreadcrumb(),
     beforeSend: (report) => rewriteReport(report),
     debug: METAMASK_DEBUG,
+    dist: isManifestV3 ? ' mv3' : 'mv2',
```

Comment:
is the space before `mv3` intentional? or the lack of space before `mv2`?

Reply:
Good catch, definitely not intentional

```suggestion
    dist: isManifestV3 ? 'mv3' : 'mv2',
```